components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3836,6 +3836,17 @@ where
                         "target region is not found, drop proposals";
                         "region_id" => region_id
                     );
+                    // Invoking callback can release txn latch, if it's still leader, following
+                    // command may not read the writes of previous commands and break ACID. If
+                    // it's still leader, there are two possibility that mailbox is closed:
+                    // 1. The process is shutting down.
+                    // 2. The leader is destroyed. A leader won't propose to destroy itself, so
```

Comment:
Looks like the `raftstore.allow-remove-leader` config is not remove yet

Reply:
I will remove the config in another PR.

Reply:
See #10384.